.gitignore

```diff
@@ -1,2 +0,0 @@
-.DS_Store
-artists.json
```

Comment:
I'm pretty sure this is supposed to be ignored so that the gen_artists.py works properly. So again, DON'T DELETE IT!